### DISCUSSION TITLE
Wiki에 넣은 내용

### DISCUSSION THREAD

**Discussion by devnproyj22** [weaverse-techtide/weaverse-backend#99] [General]
현재 Wiki에 있는 내용입니다.

## 1. Weaverse 바로가기 링크
- github 저장소 (레포지토리 목록)
- 프로젝트 소개 문서 (main README)
## 3. Backend 기술 스택
- 백엔드 프레임워크
- 데이터베이스
- API 개발
- API 문서화
- 인증 및 권한
- 테스팅
- 보안
- 배포
- 클라우드(파일 저장용)
- CI/CD
- 코드 품질
- 환경 관리
- 기능 명세
## 4. 버전 관리
- psycog2와 psycopg 버전 차이를 기술

- - -
다음은 Project의 글을 참고하여 추가해볼만한 내용입니다.
# 팀워크
## shared link
- Progress : 오름캠프 - Google Sheet
-  UI & API : Weaverse - Figma
-  API : Weaverse - Notion
-  ERD : Weaverse -ERDCloud
 ## code convention
- - -

혹시 여기서 추가하거나 뺄만한 내용이 있으면 알려주세요

**Comment by Yoo117** [comment]
도움이 될 git command도 몇가지 추가하는 편도 좋을 것 같습니다. 특히 dev에서 필요한 파일만 pull 하는 기능도 기입해두면 좋을 것 같습니다. 저도 그부분이 아직 정리가 안돼서요.

README에 올릴 내용과 겹치지 않게 구분을 지을 필요도 있어 보입니다. Wiki에는 팀원들에게, 혹은 나중에 추가로 들어올 기여자를 상정해서 도움이 될 내용들 위주로 작성하는 편이 좋을 것 같습니다.

**Comment by devnproyj22** [comment]
README.md
* 프로젝스 소개하는 내용
* 독자: 프로젝트에 관심 있는 사람

Wiki
* 협업 과정
* 뉴팀원에게 공유할 내용
* 독자: 뉴팀원, 협업을 어떻게 했는지 알고픈 사람
* 내용: 선택한 기술 스택과 그 이유, Ground Rule, Shared Link, 등....
